src/main/webapp/js/user-page-loader.js

```diff
@@ -91,6 +98,22 @@ function buildMessageDiv(message) {
   return messageDiv;
 }
 
+/** Provides link to URL */
+function buildLanguageLinks(){
+  const userPageUrl = '/user-page.html?user=' + parameterUsername;
+  const languagesListElement  = document.getElementById('languages');
+  languagesListElement.appendChild(createListItem(createLink(
```

Comment:
Did you forget to define `createListItem` and `createLink`? Or what is happening here?

In any case, a lot of repetition.
define a list or a map of languages, something like
```
const languages = [['English', 'en'], ['Chinese', 'zh'], ['Hindi', 'hi']...]
```
and then iterate over it in the loop, creating the needed links.

Reply:
Sounds good! 
The functions were from a different file, navigation-loader.js.